Tasks/ServiceFabricDeployV1/Tests/L0.ts

```diff
@@ -63,5 +63,29 @@ describe('ServiceFabricDeploy Suite', function () {
         it('Register application type should retry', (done) => {
             psr.run(path.join(__dirname, 'RegisterApplicationTypeShouldRetry.ps1'), done);
         })
+        it('Unregister application type should retry till success', (done) => {
+            psr.run(path.join(__dirname, 'UnregisterApplicationTypeShouldRetryTillSuccess.ps1'), done);
+        })
+        it('Unregister application type should retry', (done) => {
+            psr.run(path.join(__dirname, 'UnregisterApplicationTypeShouldRetry.ps1'), done);
+        })
+        it('Create application type should retry till success', (done) => {
+            psr.run(path.join(__dirname, 'CreateApplicationShouldRetryTillSuccess.ps1'), done);
```

Comment:
was this file already present

Reply:
Yes, they were part of previous PR. However, during conflict resolution, these changes were reverted in that PR.